src/parser/jobs/dnc/modules/FeatherGauge.tsx

```diff
@@ -0,0 +1,116 @@
+import {t} from '@lingui/macro'
+import {Plural, Trans} from '@lingui/react'
+import Color from 'color'
+import _ from 'lodash'
+import React, {Fragment} from 'react'
+
+import TimeLineChart from 'components/ui/TimeLineChart'
+import ACTIONS from 'data/ACTIONS'
+import JOBS from 'data/JOBS'
+import Module, {dependency} from 'parser/core/Module'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+
+import styles from './DNCGauges.module.css'
+
+const FEATHER_GENERATORS = [
+	ACTIONS.REVERSE_CASCADE.id,
+	ACTIONS.FOUNTAINFALL.id,
+	ACTIONS.RISING_WINDMILL.id,
+	ACTIONS.BLOODSHOWER.id,
+]
+
+const FEATHER_CONSUMERS = [
+	ACTIONS.FAN_DANCE.id,
+	ACTIONS.FAN_DANCE_II.id,
+]
+
+const FEATHER_GENERATION_CHANCE = .5
+const MAX_FEATHERS = 4
+
+export default class FeatherGauge extends Module {
+	static handle = 'feathergauge'
+	static title = t('dnc.feather-gauge.title')`Feather Gauge`
+
+	@dependency private suggestions!: Suggestions
+
+	private feathersConsumed = 0
+	private avgGenerated = 0
+	private history: any[] = []
```

Comment:
Is this `any` avoidable? Ideally it should never be used.

...wonder if that's something i can add to linting...